docs/GRANTS.md

```diff
@@ -1,6 +1,74 @@
 # Grants General Documentation
 
-## Rounds 6+
+## Round 7
+
+In round 7 we introduced the option to checkout on L2 using [zkSync](https://zksync.io/).
+This means there are now three possible checkout flows&mdash;the Bulk Checkout flow introduced in
+Round 6, and two zkSync checkout flows.
+
+### Architecture Background
+
+Before describing these two flows, we'll review how zkSync accounts work at a high level. When
+logging in to your zkSync account, such as at https://wallet.zksync.io/, you are asked to sign a
+message. That signature is used to generate a private key, and each transfer then requires two
+signatures&mdash;one from this private key, and as an additional security measure, one more from
+your regular web3 wallet. This is great from a security perspective, but not so great from a UX
+perspective. Requiring one signature for each transfer can be a pain if you want to send a lot of
+transfers, so you'd be doing a lot of clicking when checking out with large carts. 
+
+To remedy this, we ask you to login to a "Gitcoin zkSync" account. This account is created by
+asking you to sign a different message from the regular zkSync message, and this gives us an L1 account
+that the Gitcoin app has the private key to. We then login to zkSync with *this account*, and now
```

Comment:
Great write up! I would perhaps clarify this a bit, something like:

"this generates an L1 account that the Gitcoin frontend can use to sign transactions directly. The private key to this account comes from the signature, and is never stored or transmitted anywhere."

Reply:
That sounds good! Updated in 7ff3ef4